src/containers/BooksList/BooksList.js

```diff
@@ -1,15 +1,24 @@
-import { useState } from 'react';
+import React, { useEffect, useState } from 'react';
 import { useSelector } from 'react-redux';
 import Book from '../../components/book/index';
+import { REMOVE_BOOK } from '../../actions';
+import store from '../../reducers/index';
 
 const BooksList = () => {
-  const [Books] = useState(useSelector(((state) => state)));
+  const [Books, setBooks] = useState([]);
+  const list = useSelector(((state) => state.BookReducer));
+
+  useEffect(() => {
+    setBooks(list);
+  }, [list]);
   return (
     <table>
       <tbody>
-        {Books.map((book) => (
-          <Book obj={book} key={book.id} />
-        ))}
+        {
```

Comment:
- [ ] In the Booklist.js component you are passing down a button html element as a prop. which is definitely a bad practice. Props are supposed to be a primitive data type, an object, a function, or an array. not a whole HTML element. 
the bookList component should pass a handleRemove function to the book component. 
the book component should use said function when a user clicks on the delete button. 
Please make sure to fix this.